### ISSUE TITLE
Intercom UI can be open without Intercom access

### ISSUE THREAD

**Issue by jonpas** [IDI-Systems/acre2#786]
**Arma 3 Version:** `1.94` (stable)
**CBA Version:** `3.12.0` (stable)
**ACRE2 Version:** `2.7.0` (stable)

**Mods:**
```
- CBA_A3
- acre2
- ace
- Unsung
```

**Description:**
- Vehicle with a master station entry without access to an intercom can still open the Intercom UI. While that is probably meant to be like that, it would probably make more sense to prevent that, to signify configuration issues for the developer and making it clearer for the player that there is no active intercom for him.

**Steps to reproduce:**
- Have a vehicle with a master station entry that has no access to intercom (prints warning in RPT).
  - Test case: Unsung Hueys - master station on `commander`. Seems to be caused by inheritance due to different crew positions.
- Sit in that seat with a master station without intercom access.
- Open Intercom UI.
- Observe it can be open but it's kind of pointless since there shouldn't be one.

**Where did the issue occur?**
- All

**Placed Modules:**
- None

**RPT log file:**
```
16:21:48 [ACRE] (sys_intercom) WARNING: Vehicle type B Alpha 1-2:1 REMOTE has a master station entry (commander) that has no access to that intercom. Ignoring master positions for intercom network intercom_1 idi\acre\addons\sys_intercom\fnc_configIntercom.sqf:94
```

**Comment by TheMagnetar** [comment]
We have to differentiate between the two things. A master station provides functionality in terms of activating accent feature, configure radio access and so on. A master station is used in combination with a normal station (FFCS), and the latest is the one providing intercom capabilities, not the master station.

**Comment by jonpas** [comment]
Yes, only reason I provided that in log is because that's the only ACRE thing that showed up apart from regular loading messages.

However, I tested this further now and it seems to be a bigger problem, unsure if with just configuration right now, I will test further in the future.
